tests/test_ds.py

```diff
@@ -3,6 +3,7 @@
 import dagmc_stats.DagmcStats as ds
 import pandas as pd
 
+entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
```

Comment:
We should make `entity_types` a class variable of `DagmcStats` and then we can refer to it in the tests as `ds.entity_types`